src/test_l1_dsFPD.c

```diff
@@ -3044,7 +3021,7 @@ void test_l1_dsFPD_negative_dsFPSetLEDState(void)
     UT_ASSERT_EQUAL_FATAL(result, dsERR_NONE);
 
     // Step 03: Set all valid indicators to dsFPD_STATE_ON using dsSetFPState()
-    count = UT_KVP_PROFILE_GET_UINT32("dsFPD/Number_of_Indicators");
+    count = UT_KVP_PROFILE_GET_UINT8("dsFPD/Number_of_Indicators");
     enableFPDIndicators(count);
 
     // Step 04: Call dsFPSetLEDState() with an invalid LED state
```

Comment:
disable all the FPD Indicator before Term

Reply:
Done